services/services.rst

```diff
@@ -23,36 +23,18 @@ SuMIRe/PFS PBworks wiki
 
 "SuMIRe/PFS PBworks" is the only official wiki system for PFS. 
```

Comment:
It sounds like as if PBworks is one of the internal services.

Reply:
added pbworks into sample of external service (in the top part) #e6ec8b0